rackcity/views/asset_views.py

```diff
@@ -506,16 +511,19 @@ def save_power_connections(asset_data, asset_id, change_plan=None):
                     left_right=power_connection_data['left_right'],
                     port_number=power_connection_data['port_number']
                 )
+                pdu_port = pdu_port_master
```

Comment:
it seems like we prefer these johnsons to be "live" not "master" so maybe change to make consistent?

Reply:
johnsons heh